props/saber_fett263_buttons.h

```diff
@@ -183,13 +255,13 @@ SaberFett263Buttons() : PropBase() {}
         swing_blast_ = true;
         hybrid_font.SB_Blast();
         return true;
-	
+
       case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_ON):
-	if (swing_blast_) {
+        if (swing_blast_) {
 	  SaberBase::DoBlast();
```

Comment:
indent two steps

Reply:
Is it possible it looks different for me? Because I'm lost- in my view "if" is indented two steps from "case" and "SaberBase" is indented two steps from "if" but this the comment view is showing them lined up so I'm totally lost on these indentations.   I don't see anything wrong with line 261 when I'm in Edit view.

Reply:
What editor are you using?
Do you have tabs in the code?
Tabs can look different in different editors.
Basically, tabs are evil.

Reply:
I've just been editing on github.com website and it looks all fine on my end, I use space bar because the tab jumps multiple spots.  But this one in particular when I view it on github.com in edit mode it's lined up fine but the comment isn't so I'm just lost

Reply:
I think what we're going to do is to just fix everything *except* the indentation.
I'll fix that afterwards.
However, for next time, we're going to have to figure out a way where you can see and edit the indentation correctly.